docs/how-to/uc-image-creation.rst

```diff
@@ -28,12 +28,24 @@ Create the model assertion
 The model assertion is a digitally signed document that describes the content of the Ubuntu Core image.
 Read the `model assertion`_ documentation before continuing.
 
-Below is an example model assertion in YAML, describing a `core22` Ubuntu Core
-image:
+Below are example model assertions, describing the Ubuntu Core image content for most recent releases:
```

Comment:
Nitpick here.
Does "most recent releases" here mean it includes "most of the recent releases"? Or "the most recent release"?

I would remove "most" and just say "...describing the Ubuntu Core image content for recent releases" instead.